### PR TITLE
SC CLI edits

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

- Consolidated Data Center options information
- Misc. edits for style and consistency

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/cli/sauce-connect-proxy.md [hunk 1]
```diff
@@ -149,21 +150,36 @@ __Shorthand__: `-t`
...
-Effective with Sauce Connect Proxy version 4.7.0, we recommend using [`--region`](/dev/cli/sauce-connect-proxy/#--region) instead. `--rest-url` will eventually be deprecated. Download the latest SC version [here](/secure-connections/sauce-connect/installation/).
+As this option will soon be deprecated, we recommend using the [`--region`](/dev/cli/sauce-connect-proxy/#--region) flag, compatible with Sauce Connect versions 4.7.0 and above.
 :::
 
-__Description__: Sets your [Sauce Labs regional data center REST API URL](#data-center-endpoints) (e.g., EU-Central, US-West).<br/>
+__Description__: Sets the [Sauce Labs data center endpoint URL](/basics/data-center-endpoints) of the location where the device you're testing on is hosted.<br/>
 __Enviroment variable__: `SAUCE_REST_URL`<br/>
 __Shorthand__: `-x`
 
+```bash
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:160]: The endpoint to connect to **EU-Central** is listed twice (https://eu-central-1.saucelabs.com/rest/v1). Should it be replaced with the endpoint for **US-East** (https://us-east-1.saucelabs.com/rest/v1) + should we add the endpoint for **US-West** (https://api.us-west-1.saucelabs.com/rest/v1) as well?

File: docs/dev/cli/sauce-connect-proxy.md [hunk 2]
```diff
@@ -44,28 +44,29 @@ __Shorthand__: `-k`
...
 ---
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
-__Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Description__: Sets your Sauce Labs [data center region](#data-center-endpoints) (e.g., `us-west`, `eu-central`, `apac-southeast`).<br/>
+__Default__: If you don't specify a Data Center, Sauce Connect will default to `us-west`. <br/>
 __Enviroment variable__: `SAUCE_REGION`<br/>
 __Shorthand__: `-r`
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:62]: We could add code examples with the required options for each region, like you did for the --rest-url flag below. For example: `./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -r us-west`, etc.

File: docs/dev/cli/sauce-connect-proxy.md [hunk 3]
```diff
@@ -303,7 +319,7 @@ __Shorthand__: `-P`
...
 
-Sauce Connect Proxy's `--auth` flag will only send the header Authorization with a type of 'Basic'. If a resource responds with the header WWW-Authenticate of a type any other than 'Basic,' your authentication will fail and return a non-200 HTTP response. HTTP Header Injection is disabled for SSL domains that are not re-encrypted by Sauce Connect Proxy, which means performing basic authentication in this way is disabled for all HTTPS domains passed to `--no-ssl-bump-domains` argument.<br/>
+This flag will only send the header Authorization with a type of "Basic". If a resource responds with the header WWW-Authenticate of a type any other than "Basic," your authentication will fail and return a non-200 HTTP response. HTTP Header Injection is disabled for SSL domains that are not re-encrypted by Sauce Connect Proxy, which means performing basic authentication in this way is disabled for all HTTPS domains passed to `--no-ssl-bump-domains` argument.<br/>
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:322]: Based on my current understanding of our style guide, we should use replace **"Basic".** with **"Basic."** in the first sentence. 

Although personally, I prefer the use of punctuation marks _outside_ of the quotations marks as I find it makes it easier for readability. Our style guide isn't explicit about this -- Can we discuss as a team?

**Review by lysannep — CHANGES_REQUESTED** [review]
Reviewed. I added a few comments/suggestions.

File: docs/dev/cli/sauce-connect-proxy.md [hunk 4]
```diff
@@ -44,28 +44,29 @@ __Shorthand__: `-k`
...
+:::
 
 ---
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
-__Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Description__: Sets your Sauce Labs [data center region](#data-center-endpoints) (e.g., `us-west`, `eu-central`, `apac-southeast`).<br/>
+__Default__: If you don't specify a Data Center, Sauce Connect will default to `us-west`. <br/>
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/dev/cli/sauce-connect-proxy.md:60]: @lysannep I was going to run that past you guys! Appreciate you mentioning it. I will create a separate ticket for this task.

File: docs/dev/cli/sauce-connect-proxy.md (hunk 1 above)
Comment (kimsauce) [reply docs/dev/cli/sauce-connect-proxy.md:160]: good catch! I will message you separately about this.

**Review by lysannep — APPROVED** [review]
Looks good!

**Pull request merged**